src/hardware/pcspeaker_discrete.h

```diff
@@ -52,7 +52,7 @@ class PcSpeakerDiscrete final : public PcSpeaker {
 	static constexpr char device_name[] = "PCSPEAKER";
 	static constexpr char model_name[]  = "discrete";
 
-	static constexpr float pwm_scalar = 0.5f; // accomodate a full doubling
+	static constexpr float pwm_scalar = 0.72f; 
```

Comment:
That seems oddly specific... not just 0.7, but 0.7**2**! 😂 I think it warrants a comment? Was it derived from a decibel scale factor perhaps?

Reply:
Good point; added a comment and reference to the hardware capture.